web-share/share-null-manual.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share null and undefined values</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Expect null to be converted into the string 'null'. (This isn't good
+        // practice, but it's how Web IDL specifies conversion.) On the other
```

Comment:
Gotcha! While I agree null vs. undefined isn't the most important thing in the world, it's the kind of thing that cause hard to reproduce issues that developers end up wasting a huge amount of time on and hating the platform for. So making sure sure we're consistent here seems like a good use of ones time.